File: wiseclientlibrary/src/main/java/com/wiseclient/script/UserActionInterface.java
package com.wiseclient.script;

/**
 * Created by kuanghaochuan on 2017/7/16.
 */

public interface UserActionInterface {
    void actionViewClick(int x, int y);

    void actionViewMove(int oldX, int oldY, int newX, int newY);

    void actionBackPress();

    void actionMenuPress();

    void actionHomePress();
}

File: wiseclientlibrary/src/main/java/com/client/script/UserActionInterface.kt
package com.client.script

/**
 * Created by kuanghaochuan on 2017/7/16.
 */

interface UserActionInterface {
    fun actionViewClick(x: Int, y: Int)

    fun actionViewMove(oldX: Int, oldY: Int, newX: Int, newY: Int)

    fun actionBackPress()

    fun actionMenuPress()

    fun actionHomePress()

    fun actionKeyUpPress()

    fun actionKeyDownPress()

    fun actionKeyLeftPress()

    fun actionKeyRightPress()

    fun actionKeyEnterPress()

    fun actionKeyBackPress()
}
